sqlmesh/dbt/adapter.py

```diff
@@ -290,12 +290,14 @@ def execute(
             self.engine_adapter.fetchdf if fetch else self.engine_adapter.execute  # type: ignore
         )
 
-        expression = parse_one(sql, read=self.engine_adapter.dialect)
+        # parse SQL in its dialect
+        expression = parse_one(sql, read=self.dialect)
```

Comment:
As per convo in slack: I believe that `self.dialect` and `self.engine_adapter.dialect` are actually the same dialect since `self.dialect` is derived from the type of the dbt target.

Reply:
Yeah, these are the same value. The constructor sets dialect to engine_adapter.dialect. I guess it isn't bad to use dialect here, but we should be consistent throughout then. I think I saw other places where we do engine_adapter.dialect.